command/agent.go

```diff
@@ -25,32 +29,29 @@ var Agent = cli.Command{
 		client := &http.Client{
 			Timeout: time.Second * 3,
 		}
-		req, err := http.NewRequest("GET", "http://localhost:8000/backups", nil)
-		if err != nil {
-			log.Fatal("Error reading request. ", err)
-		}
-		auth.AddAuthHeader(req, tokenResponse)
 
-		resp, err := client.Do(req)
-		if err != nil {
-			log.Fatal("Error reading response. ", err)
-		}
-		defer resp.Body.Close()
+		backups := map[string]actions.Backup{}
+		crons := map[string]*cron.Cron{}
 
-		body, err := ioutil.ReadAll(resp.Body)
-		if err != nil {
-			log.Fatal("Error reading body. ", err)
-		}
+		cr := cron.New()
+		cr.AddFunc("* * * * *", func() {
```

Comment:
Can we run this function immediately when the agent starts, as well as every minute? I'd probably extract the anonymous function to a named function, then call it.

```go
check() // pick a better name than me
cr.AddFunc("* * * * *", check)
```